homeassistant/components/history_stats/sensor.py

```diff
@@ -77,25 +79,27 @@ def exactly_two_period_keys(conf):
 # noinspection PyUnusedLocal
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the History Stats sensor."""
-
     setup_reload_service(hass, DOMAIN, PLATFORMS)
 
     entity_id = config.get(CONF_ENTITY_ID)
-    entity_state = config.get(CONF_STATE)
+    entity_states = config.get(CONF_STATE)
     start = config.get(CONF_START)
     end = config.get(CONF_END)
     duration = config.get(CONF_DURATION)
     sensor_type = config.get(CONF_TYPE)
     name = config.get(CONF_NAME)
 
+    if isinstance(entity_states, str):
+        entity_states = [entity_states]
+
```

Comment:
With the above adjustment, this isn't needed anymore, as the result is always a list.